integration/test_collection_filter.py

```diff
@@ -313,7 +305,7 @@ def test_filters_contains(
             Property(name="uuid", data_type=DataType.UUID),
         ],
     )
-    if not collection._connection._weaviate_version.is_at_least(1, 23, 0) and skip:
+    if skip:
         pytest.skip("not supported in this version")
```

Comment:
this can also go away?

Reply:
Yeh wasn't sure about this one so left, will remove completely now

Reply:
can't remember why it is there, but we moved past "not supported in this version" :)